### PR TITLE
Added reset option for perdown

### PR DESCRIPTION
**Pull request by Sylphony**
#33 

Usage in HTML:
```
<div id="timer" data-reset></div>
```

Or in JS:
```
$("#timer").percircle({
    perdown: true,
    reset: true
});
```

Demo:
perdown

### CODE REVIEW

**Comment by Sylphony** [comment]
I also moved some of the timer code to separate functions so they become reusable.  Name-wise, hopefully they are intuitive enough.

File: src/js/percircle.js [hunk 1]
```diff
@@ -123,38 +123,67 @@
                 }, 1000);
             } else if(perdown) {
                 var secs = percircle.attr('data-secs') || options.secs;
-				var timeUpText = percircle.attr('data-timeUpText') || options.timeUpText;
-				if (timeUpText.length > 8) timeUpText='the end';
-                
-				var counter=setInterval(timer, 1000); 
+                var timeUpText = percircle.attr('data-timeUpText') || options.timeUpText;
+                var reset = percircle[0].hasAttribute('data-reset') || options.reset;
```
Comment (toubou91) [inline src/js/percircle.js]: Sorry, just curious, why this one doesn't retrieve the `reset` attribute as it's done in the rest of the cases (i.e. `percircle.attr('data-reset')`

Comment (Sylphony) [inline src/js/percircle.js]: `.attr()` gets the value of `data-reset`, in which case there is none (meaning it is undefined).  Reset is a boolean so if you don't want to make the perdown element reset, simply don't put the attribute in the first place.

Comment (toubou91) [inline src/js/percircle.js]: Oh I just took a quick look at it in the morning and I thought you put something like `data-reset="true"` in the html usage example above. Cool!

**Comment by toubou91** [comment]
Nice one @Sylphony , thank you!!!

**Comment by Sylphony** [comment]
No problem, thanks for the merge.  Keep up the awesome work!

**Pull request merged**